### DISCUSSION TITLE
Error! Binaries missing/corrupted.

### DISCUSSION THREAD

**Discussion by smithyjames391** [jely2002/youtube-dl-gui#520] [Q&A]
I'm reviving this again, because it hasn't really been answered since its first instance in 2022 and definitely hasn't been fixed. This shows up in Win 10 and 11 anytime I try to download a video. I have no antivirus other than Defender (which doesn't see this as an issue) and even when I disable, nothing changes. Are there any work arounds people have found?

The Error: "Error! Binaries missing/corrupted." and video won't download. Kind of a waste of space on my computer if it can't do the one thing it's designed for.

**Comment by sky-limits** [comment]
Having the same issue!

**Comment by gitTerracide** [comment]
FIXED: Had same issue...

Ran the installer initially, and got the errors in question. I did not uninstall the original version. I downloaded the exe below, extracted it to a different location and ran the app exe. It then downloaded the files it needed. Worked after that.



Steps I took to get it working:
1. Went to: https://github.com/StefanLobbenmeier/youtube-dl-gui/releases/tag/v2.4.10

2. Downloaded: Open-Video-Downloader-2.4.10.exe -- 51.3 MB --Apr 18, 2023 ([Open-Video-Downloader-2.4.10.exe](https://github.com/StefanLobbenmeier/youtube-dl-gui/releases/download/v2.4.10/Open-Video-Downloader-2.4.10.exe))

3. Extracted the exe with winrar to its own folder (I did not attempt to open the downloaded file): C:\\...\\_installs\Open-Video-Downloader-2.4.10

4. Opened/executed the extracted file: 'Video Downloader.exe'.

5. Made sure the app could connect to the Internet.

6. Watched it download the needed files.

7. Started using it. :)

**Reply by betocanal** [reply]
Freaking genius! You saved me, thanks buddy

**Reply by rodneykipkoech02-alt** [reply]
Thanks alot for this i was almost giving up

**Reply by neiltheneil** [reply]
this didnt work for me.

**Comment by jely2002** [comment]
Hey all, this no longer happens in the v3 versions of the app. Just cleaning up some discussions.
